modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -0,0 +1,161 @@
+= Troubleshooting
+------------------
+
+Silverblue is a radically new way of deploying and managing your
+desktop operating system, so you may ocassionally run into problems
+while going through your day to day.  Below are some of the more
+common problems reported and any workarounds for those problems.
+
+"Forbidden base package replacements"
+--------------------------------------
+https://github.com/projectatomic/rpm-ostree/issues/415
+
+This can happen when a package that is being layered has a
+dependency on a package that is in the base OS.  In the
+problematic case, the layered packcage requires a newer version
+of the dependent package which is not available in the base OS.
+
+In most cases, waiting for a newer `ostree` compose will resolve
```

Comment:
Since here we're not talking about `/usr/bin/ostree` I'd write "OSTree compose" or maybe just "base image".

Reply:
Fixed this and a few other places where it existed